public/app/plugins/panel/xychart/v2/XYChartTooltip.tsx

```diff
@@ -97,8 +98,9 @@ export const XYChartTooltip = ({ dataIdxs, seriesIdx, data, xySeries, dismiss, i
 
   if (isPinned && seriesIdx != null) {
     const links = getDataLinks(yField, rowIndex);
+    const actions = getFieldActions(data[0], yField, replaceVariables);
```

Comment:
🫤

Reply:
about `data[0]`?

you should be able to look up the frame related to the y field by searching `data`. something like

```ts
const frame = data.find(frame => frame.fields.includes(yField));
```